src/screens/ProfileScreen/elements/BottomSheetBio.js

```diff
@@ -39,6 +44,10 @@ const BottomSheetBio = React.forwardRef((props, ref) => {
   );
 });
 
+BottomSheetBio.propTypes = {
+  handleSave: PropTypes.func
+};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

It's great that you've added PropTypes for type checking, but it seems like you've only added it for `handleSave`. It would be beneficial to add PropTypes for all props used in this component to ensure they are being used correctly and to provide clear documentation for other developers.

```diff
  BottomSheetBio.propTypes = {
+   isOtherProfile: PropTypes.bool,
+   value: PropTypes.string,
+   error: PropTypes.string,
+   isLoadingUpdateBio: PropTypes.bool,
    handleSave: PropTypes.func
  };
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->